definitions/tools/grolar.cwl

```diff
@@ -79,7 +79,7 @@ requirements:
              output$geneA.id <- gsub(".\\d+$", "", output$geneA.id, perl = TRUE)
              output$geneB.id <- gsub(".\\d+$", "", output$geneB.id, perl = TRUE)
              cat(paste0("Writing out table: ", JSON_file, "_fusions_filt_sorted.txt", "\n"))
-             write.table(output[,c(-3,-4)], file = paste0(JSON_file, "_fusions_filt_sorted.simple.txt"), quote = FALSE, sep = "\t", row.names = FALSE, col.names = TRUE)
+             write.table(output[,c(-3,-4)], file = paste0(JSON_file, "_fusions_filt_sorted.txt"), quote = FALSE, sep = "\t", row.names = FALSE, col.names = TRUE)
```

Comment:
This function never gets called, as far as I can tell.  The lone call is to the `GetFusionz_and_namez` function, which does include the proper suffix.

Reply:
I've noticed that before as well, should we just remove it entirely? We could include it here or in my PR #781 which already has the fix for the file not found error

Reply:
I missed the other function not being called, this line though in GenFusionz _and_namez is called and produces the error
https://github.com/genome/analysis-workflows/blob/ddd748516b25256a461ea9277303406fa2759b00/definitions/tools/grolar.cwl#L103

Reply:
I agree that it's confusing, but I think I'd like to minimize the differences from the maintainer's grolar script, so that we don't have to do a bunch of work if he releases new bugfixes or something. Maybe we could add comment or two?